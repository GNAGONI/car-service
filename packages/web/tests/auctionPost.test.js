import React from 'react'
import renderer from 'react-test-renderer'
import AuctionPost from '../components/carAuctionResults/auctionPost'

it('renders correctly', () => {
  const tree = renderer.create(<AuctionPost />).toJSON()

  expect(tree).toMatchSnapshot()
})
