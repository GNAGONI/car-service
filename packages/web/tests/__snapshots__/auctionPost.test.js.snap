// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="post-auction  col-lg-4 col-md-6 col-sm-12"
>
  <div
    className="holder"
  >
    <div
      className="img-thumbnail"
    >
      <img
        alt="Default car"
        height="228"
        src=""
        width="380"
      />
    </div>
    <div
      className="frame"
    >
      <h3>
        <a
          href={
            Object {
              "pathname": "/car-auctions/car-auction-details",
              "query": Object {
                "id": undefined,
              },
            }
          }
        />
      </h3>
      <span
        className="price"
      >
        Â£
         
        <span
          className="text"
        >
          ( 
           bids )
        </span>
      </span>
      <time
        className="time"
      >
        00 sec
      </time>
      <div
        className="btn-block extend"
      >
        <button
          className="btn btn-primary"
          disabled={false}
          onClick={[Function]}
          primary={true}
          type="button"
        >
          View this auction
        </button>
      </div>
    </div>
  </div>
</div>
`;
